src/core/IO/Handle.pm6

```diff
@@ -1,8 +1,7 @@
-my class IO::Path { ... }
 my class Proc { ... }
 
 my class IO::Handle {
-    has $.path;
```

Comment:
I disagree with this rename. I use `IO::Handle` based abstractions for http, pipes, etc which all have a path/uri representation but are not files. It also seems to add some needless complexity to the constructor which now can seemingly take both `path` and `file`.